docs/medical-api/getting-started/quickstart.mdx

```diff
@@ -168,9 +186,9 @@ To query for a Patient's available `Documents`, you can either:
 
 ### Query for Documents using the API
 
-1. Start an asynchronous [Document Query](/medical-api/api-reference/document/start-document-query) across HIEs.
-1. [List Documents](/medical-api/api-reference/document/list-documents) to get the available document
-references returned from HIEs.
+1. Start an asynchronous [Document Query](/medical-api/api-reference/document/start-document-query) across HIEs;
+1. Then, either poll the API to [List Documents](/medical-api/api-reference/document/list-documents) to get the available document
+   references returned from HIEs - or wait process the inbound [Webhook message](/medical-api/more-info/webhooks) which will be sent if you have Webhooks configured.
```

Comment:
Maybe:

```md
2. Receive data from Metriport using the integration approach implemented on [Integrate your app with Metriport](#integrate-your-app-with-Metriport)
```

And could move the links/details we have here to there

Reply:
To prevent cognitive overhead with introducing endpoints out-of-order I'll leave the links/details here, but change the order to highlight the WH approach as suggested